src/Board.coffee

```diff
@@ -40,11 +40,11 @@ getRandomTile = ->
     pit: 35
     coin: 30
   insane =
-    dirt: 25
-    pit: 65
-    coin: 10
+    dirt: 50#25
```

Comment:
what was the reasoning behind the change to the insane difficulty change percentages?